.github/scripts/changed-modules.sh

```diff
@@ -0,0 +1,151 @@
+#   Expected output: [arangodb], only
+#
+# 5. Three files in three different modules are modified:
+#   ALL_CHANGED_FILES="packages/modules/arangodb/a.txt packages/modules/cassandra/b.txt packages/modules/chromadb/c.txt" ./.github/scripts/changed-modules.sh
+#   Expected output: [arangodb, cassandra, chromadb]
+#
+# 6. Core files and module files are modified:
+#   ALL_CHANGED_FILES="packages/testcontainers/a.txt packages/modules/chromadb/b.txt" ./.github/scripts/changed-modules.sh
+#   Expected output: all modules, as the core has been modified
+#
+# 7. This script is modified:
+#   ALL_CHANGED_FILES=".github/scripts/changed-modules.sh" ./.github/scripts/changed-modules.sh
+#   Expected output: all modules, as the build script has been modified
+#
+# 8. A .github file is modified:
+#   ALL_CHANGED_FILES=".github/release-drafter.yml" ./.github/scripts/changed-modules.sh
+#   Expected output: []
+#
+# 9. A excluded module is modified:
+#   ALL_CHANGED_FILES="packages/modules/couchbase/a.txt" ./.github/scripts/changed-modules.sh
+#   Expected output: []
+#
+# There is room for improvement in this script. For example, it could detect if the changes applied to the docs or the .github dirs, and then do not include any module in the list.
+# But then we would need to verify the CI scripts to ensure that the job receives the correct modules to build.
+
+# ROOT_DIR is the root directory of the repository.
+readonly ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)
+
+# define an array of directories that won't be included in the list
+readonly excluded_modules=(".devcontainer" ".vscode" ".husky" "docs" ".github/ISSUE_TEMPLATE")
+
+# define an array of files that won't be included in the list
+# Get all files in the root directory except package.json and package-lock.json
+# Create array of excluded files by finding all files in root dir except package.json and package-lock.json
+excluded_files=("${ROOT_DIR}/.github/release-drafter.yml")
+while IFS= read -r file; do
+  excluded_files+=("\"${file}\"")
+done < <(find "${ROOT_DIR}" -maxdepth 1 -type f -not -name "package.json" -not -name "package-lock.json")
+
+# define an array of modules that won't be part of the build
+readonly no_build_modules=("couchbase")
+
+# modules is an array that will store the paths of all the modules in the repository.
+modules=()
+
+# Find all package.json files in the repository, building a list of all the available modules.
+# The list of modules is stored in the modules array, but the testcontainers-node module is excluded
+# as it is not a module.
+for packageJSONFile in $(find "${ROOT_DIR}" -name "package.json" -not -path "*/node_modules/*"); do
+    name=$(basename "$(dirname "${packageJSONFile}")")
+    if [[ "${name}" != "testcontainers-node" ]]; then
+        modules+=("\"${name}\"")
+    fi
+done
+# sort modules array
+IFS=$'\n' modules=($(sort <<<"${modules[*]}"))
+unset IFS
```

Comment:
Duplicate

Reply:
Good catch, removed in [39b904d](https://github.com/testcontainers/testcontainers-node/pull/918/commits/39b904dc66b051a47aefd520091f78f40c82de87)